contrib/cirrus/spoof_travis.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/test/ctr.bats b/test/ctr.bats
+index d2945c3da..bc8f39a91 100644
+--- a/test/ctr.bats
++++ b/test/ctr.bats
+@@ -1075,7 +1075,7 @@ function teardown() {
+ }
+ 
+ @test "ctr oom" {
+-	if [[ "$TRAVIS" == "true" ]]; then
++	if [[ "true" == "true" ]]; then
+ 		skip "travis container tests don't support testing OOM"
+ 	fi
+ 	start_crio
```

Comment:
Instead of using another patch, would it be simpler to just add `TRAVIS: "true"` in `.cirrus.yml` under the `env:` section?